helm/alfresco-identity-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: alfresco-identity-service
-version: 1.1.2-AUTH-259
+version: 1.1.3
```

Comment:
I think this is a large enough change to necessitate incrementing the chart version to 1.2.0 as well, potentially even 2.0.0 since we've learned it's a breaking change without an upgrade to Helm.